src/Catalyst.Cli/Program.cs

```diff
@@ -0,0 +1,114 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Autofac;
+using Catalyst.Abstractions.Cli;
+using Catalyst.Core.Lib.Kernel;
+using Catalyst.Protocol.Network;
+using CommandLine;
+
+namespace Catalyst.Cli
+{
+    internal static class Program
+    {
+        private static readonly Kernel Kernel;
+
+        internal class Options
+        {
+            [Option("network-file", HelpText = "The name of the network file")]
+            public string OverrideNetworkFile { get; set; }
+        }
+
+        static Program()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Initialize all 'static fields' inline and remove the 'static constructor'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)